tests/integration/server/main_test.go

```diff
@@ -29,10 +30,21 @@ var (
 	testArtifactsDir        string
 )
 
+//go:linkname startEtcd k8s.io/kubernetes/test/integration/framework.startEtcd
+func startEtcd() (func(), error)
```

Comment:
`framework.EtcdMain` has a check for leaking go routines which keeps tripping us up. so we should just use the `startEtcd` directly until we want to spend time hunting down those leaks unrelated to our code here in this repository.

NOTE `startEtcd` is not a public method in the `framework` package, so we have to hack around.

Reply:
fixing this in https://github.com/kubernetes/kubernetes/pull/115254

Reply:
Some/all of this info is worth adding to a code comment.  I would definitely have to dig up this PR if I came across this code without knowing the context.

Reply:
will we change back to framework.EtcdMain when the fixing in kubernetes are in a new release?

Reply:
`framework.EtcdMain` YESSSS!!